helpers/config.py

```diff
@@ -73,16 +106,16 @@ def verify_settings():
     data['maps']['closest_z_alpha'] = get_setting(
         data['maps'].get('closest_z_alpha', 20),
         20,
-        lambda x: (x >= 1 and x <= 100)
+        lambda x: (1 <= x <= 100)
         )
     data['maps']['current_z_alpha'] = get_setting(
         data['maps'].get('current_z_alpha', 100),
         100,
-        lambda x: (x >= 1 and x <= 100)
+        lambda x: (1 <= x <= 100)
         )
     data['maps']['geometry'] = get_setting(
-        data['maps'].get('geometry', [100, 100, 400, 200]),
-        [100, 100, 400, 200],
+        data['maps'].get('geometry', [0, 0, 400, 400]),
```

Comment:
Putting these flush may be a contentious choice as well -- I should probably revert this change, AND make the "flush" option default to `False`.